### PR TITLE
`compiler_output` docstrings

### PR DESCRIPTION
**Pull request by epelaaez**
Docstrings for `qiskit-superstaq/qiskit_superstaq/compiler_output.py`.

### CODE REVIEW

File: qiskit-superstaq/qiskit_superstaq/compiler_output.py [hunk 1]
```diff
@@ -59,6 +75,18 @@ def __init__(
         jaqal_programs: Optional[Union[str, List[str]]] = None,
         pulse_lists: Optional[Union[List[List[List[Any]]], List[List[List[List[Any]]]]]] = None,
     ) -> None:
+        """Constructs a `CompilerOutput` object.
+
+        Args:
+            circuits: Compiled circuits.
```
Comment (bharat-thotakura) [inline qiskit-superstaq/qiskit_superstaq/compiler_output.py:81]: nit
```suggestion
            circuits: Compiled circuit or list of compiled circuits.
```

File: qiskit-superstaq/qiskit_superstaq/compiler_output.py [hunk 2]
```diff
@@ -130,13 +161,15 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
     """Reads out returned JSON from SuperstaQ API's AQT compilation endpoint.
```
Comment (bharat-thotakura) [inline qiskit-superstaq/qiskit_superstaq/compiler_output.py:161]: minor nit: Can you update SuperstaQ to Superstaq wherever applicable in the docstrings? For example:
```suggestion
    """Reads out returned JSON from Superstaq API's AQT compilation endpoint.
```

**Review by bharat-thotakura — COMMENTED** [review]
Minor docstring nits, but looks good after that!

**Comment by epelaaez** [comment]
All done @bharat-thotakura

File: qiskit-superstaq/qiskit_superstaq/compiler_output.py [hunk 3]
```diff
@@ -17,7 +17,14 @@
 
 
 def active_qubit_indices(circuit: qiskit.QuantumCircuit) -> List[int]:
-    """Returns the indices of the non-idle qubits in a quantum circuit."""
+    """Returns the indices of the non-idle qubits in a quantum circuit.
```
Comment (bharat-thotakura) [inline qiskit-superstaq/qiskit_superstaq/compiler_output.py:20]: nit:
```suggestion
    """Returns the indices of the non-idle qubits in the input quantum circuit.
```

File: qiskit-superstaq/qiskit_superstaq/compiler_output.py [hunk 4]
```diff
@@ -30,7 +37,14 @@ def active_qubit_indices(circuit: qiskit.QuantumCircuit) -> List[int]:
 
 
 def measured_qubit_indices(circuit: qiskit.QuantumCircuit) -> List[int]:
-    """Returns the indices of the measured qubits in a quantum circuit."""
+    """Returns the indices of the measured qubits in a quantum circuit.
```
Comment (bharat-thotakura) [inline qiskit-superstaq/qiskit_superstaq/compiler_output.py:40]: nit: 
```suggestion
    """Returns the indices of the measured qubits in the input quantum circuit.
```

**Review by bharat-thotakura — COMMENTED** [review]
Sorry, I forgot to include some minor nits earlier. LGTM after that and branch update.

**Comment by epelaaez** [comment]
No worries @bharat-thotakura, should be good now.

**Pull request merged**